dogs.html

```diff
@@ -0,0 +1,198 @@
+               <p>Corrum volorit iandae<br>
+                  nimaxim cum restia volor<br>
+                  reicid ut et etur sunt arum<br>
+                  rendae pla endis re ea erum,<br>
+                  qui doluptae
+               </p>
+               <button onclick="adoptButton('123.45')">Adopt</button>
+            </div>
+            <div class="grid-item">
+               <img src="images/jack-card.jpg" alt="Jack Dog" style="width:100%" onclick="viewDogDetails('Jack','Foo3','123.45')"/>
+               <h4> Jack </h4>
+               <p><strong>Cost to Adopt: </strong> $123.45</p>
+               <p>Corrum volorit iandae<br>
+                  nimaxim cum restia volor<br>
+                  reicid ut et etur sunt arum<br>
+                  rendae pla endis re ea erum,<br>
+                  qui doluptae
+               </p>
+               <button onclick="adoptButton('123.45')">Adopt</button>
+            </div>
+            <div class="grid-item">
+               <img src="images/duffy-card.jpg" alt="Duffy Dog" style="width:100%" onclick="viewDogDetails('Duffy','Foo4','123.45')"/>
+               <h4> Duffy </h4>
+               <p><strong>Cost to Adopt: </strong> $123.45</p>
+               <p>Corrum volorit iandae<br>
+                  nimaxim cum restia volor<br>
+                  reicid ut et etur sunt arum<br>
+                  rendae pla endis re ea erum,<br>
+                  qui doluptae
+               </p>
+               <button onclick="adoptButton('123.45')">Adopt</button>
+            </div>
+            <div class="grid-item">
+               <img src="images/lucas-card.jpg" alt="Lucas Dog" style="width:100%" onclick="viewDogDetails('Lucas','Foo5','123.45')"/>
+               <h4> Lucas </h4>
+               <p><strong>Cost to Adopt: </strong> $123.45</p>
+               <p>Corrum volorit iandae<br>
+                  nimaxim cum restia volor<br>
+                  reicid ut et etur sunt arum<br>
+                  rendae pla endis re ea erum,<br>
+                  qui doluptae
+               </p>
+               <button onclick="adoptButton('123.45')">Adopt</button>
+            </div>
+            <div class="grid-item">
+               <img src="images/jake-card.jpg" alt="Jake Dog" style="width:100%" onclick="viewDogDetails('Jake','Foo6','123.45')"/>
+               <h4> Jake </h4>
+               <p><strong>Cost to Adopt: </strong> $123.45</p>
+               <p>Corrum volorit iandae<br>
+                  nimaxim cum restia volor<br>
+                  reicid ut et etur sunt arum<br>
+                  rendae pla endis re ea erum,<br>
+                  qui doluptae
+               </p>
+               <button onclick="adoptButton('123.45')">Adopt</button>
+            </div>
+            <div class="grid-item">
+               <img src="images/angus-card.jpg" alt="Angus Dog" style="width:100%" onclick="viewDogDetails('Angus','Foo7','123.45')"/>
```

Comment:
If you are repeating a styling, such as adding a 100% width to all images, you might want to just move that to a class. Or do something like `.grid-item img { width: 100% }` to apply to all images in that class.